src/views/ipv/page-passport-doc-check.html

```diff
@@ -0,0 +1,43 @@
+{% extends "shared/ipv-template.html" %}
+{% set hmpoPageKey = "pagePassportDocCheck" %}
+
+{% block cta %}
+  {% from "govuk/components/button/macro.njk" import govukButton %}
+  {% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+  <form id="passportDocCheckingForm" action="ipv/page/{{pageId}}" method="POST" onsubmit="return passportDocCheckingFormSubmit()">
```

Comment:
Does this need a "/" at the start of the action? I spotted that we do in one of the njk files, but not 100% sure if its needed

Reply:
I've changed it to be consistent with the other njk files